changelogs/unreleased/release-command-dash-a.yml

```diff
@@ -0,0 +1,8 @@
+description: Added `-a` option to `inmanta release` command to commit all pending changes.
+issue-nr: 2508
```

Comment:
This doesn't seem to be the correct ticket. So you want to reference 5082?

Reply:
oh! bad copy paste!

Reply:
```suggestion
```